backend/tournesol/urls.py

```diff
@@ -126,4 +126,9 @@
         PollsRecommendationsView.as_view(),
         name="polls_recommendations",
     ),
+    path(
+        "polls/<str:name>/entities/<str:uid>/criteria_scores_distributions",
```

Comment:
Out of scope for this PR: 

This increases the need and priority for #749 @GresilleSiffle @amatissart